### PR TITLE
Update delegates.json

### PR DESCRIPTION
**Pull request by TAOVali**
"5EhvL1FVkQPpMjZX4MAADcW42i3xPSF1KiCpuaxTYVr28sux": {
        "name": "TAO-Validator.com", 
        "url": "www.tao-validator.com", 
        "description": "TAO-Validator.com is a highly secure validator on the Bittensor Network. Earn up to 4% extra APY"

### CODE REVIEW

File: delegates.json [hunk 1]
```diff
@@ -63,5 +63,10 @@
         "name": "Tensor.Exchange", 
         "url": "www.tensor.exchange", 
         "description": "Bittensor's first community OTC exchange"
-    }
+    },
+    "5EhvL1FVkQPpMjZX4MAADcW42i3xPSF1KiCpuaxTYVr28sux": {
+        "name": "TAO-Validator.com", 
+        "url": "www.tao-validator.com", 
+        "description": "TAO-Validator.com is a highly secure validator on the Bittensor Network. Earn up to 4% extra APY"
```
Comment (camfairchild) [inline delegates.json:70]: Please remove mention of APY

**Review by camfairchild — CHANGES_REQUESTED** [review]

**Comment by TAOVali** [comment]
Will do, thanks for the quick reply

**Comment by TAOVali** [comment]
Hi, could you please review the new pull request? I changed APY and used what other validators are using. Please let me know if this is alright, I would like to have our name up their asap. thank you

**Comment by camfairchild** [comment]
Hi, I'm not seeing any changes in this PR, nor any new PRs? I'll re-open when you amend this

**Comment by camfairchild** [comment]
Hi, I see the new PR #1203

**Comment by TAOVali** [comment]
I’ve made a new PR >> 1217, could we please get this online today? Thank you!

**Pull request closed without merging**